tests/foreman/destructive/test_installer.py

```diff
@@ -147,3 +147,39 @@ def test_positive_installer_certs_regenerate(target_sat):
     )
     assert result.status == 0
     assert 'FAIL' not in target_sat.cli.Base.ping()
+
+
+def test_positive_installer_puma_worker_count(target_sat):
+    """Installer should set the puma worker count and thread max without having to manually
+    restart the foreman service.
+
+    :id: d0e7d958-dd3e-4962-bf5a-8d7ec36f3485
+
+    :steps:
+        1. Check how many puma workers there are by default (9)
+        2. Change answer's file to have 5 puma workers
+        3. Run satellite-installer --foreman-foreman-service-puma-workers 5 --foreman-foreman-service-puma-threads-max 5
+
+    :expectedresults: aux should show there are only 5 puma workers after installer runs
+
+    :BZ: 2025760
+
+    :customerscenario: true
+    """
+    result = target_sat.execute('ps aux | grep -v grep | grep -e USER -e puma')
```

Comment:
```suggestion
    result = target_sat.execute('ps aux | grep -e USER -e "puma: cluster worker"')
```

Reply:
I'm just following what the BZ did https://bugzilla.redhat.com/show_bug.cgi?id=2025760. I'd prefer to keep as is.